### DISCUSSION TITLE
Not all tailwind classes available?

### DISCUSSION THREAD

**Discussion by davebb16** [ben-rogerson/twin.macro#708] [Q&A] [answered]
Hey guys, i'm pretty new to twin and there's a lot that I like about it... however, it seems like some classes arent available when i am using twin macro. In my case, i was trying to use the class 'scroll-p-4'. When i try to use this in my jsx file, it says the class doesnt exist. However, if i use '@apply scroll-p-4;' in my tailwind.css file it works fine. Any ideas? is it something wrong with my config?
Thanks in advance

**Comment by ben-rogerson** [comment, marked as answer]
The next major version of twin includes those new classes and is currently in release candidate - follow along in #679

**Reply by inquiloper** [reply]
I'm having the same problem but with outline and some other random classes. Does that mean that there's a mismatch between the classes offered by tailwind and twin macro?

**Reply by davebb16** [reply]
yea, it sounds like there is

**Comment by Sowed** [comment]
@davebb16,  I am also struggling with the same issue. Especially for custom colors defined within the tailwind CSS config. Twin macro cannot pick those up. Some other colors like `violet` also trigger errors. Currently using...

```json
    "styled-components": "5.3.5",
    "tailwindcss": "3.1.8",
    "twin.macro": "2.8.2"
```

Everything else is pure 🧡. @ben-rogerson, thank you for this package man!

---
Edits

- Saw one section of adding [`color from the config`](https://github.com/ben-rogerson/twin.macro/pull/106) but I find it more cumbersome compared to the generated color utils by tailwind.
- Tried installing `twin.macro@rc` ~ `"^3.0.0-rc.5"`, but the build failed on some of my styles.

**Reply by davebb16** [reply]
The ones in the tw config work for me within twin... Any colors that weren't available, I was able to add through there to make them usable

**Reply by davebb16** [reply]
```
    "styled-components": "5.3.6",
    "tailwindcss": "3.1.8",
    "twin.macro": "2.8.2",
```
